agents/builtin/mysql/slowlog/slowlog.go

```diff
@@ -148,7 +149,7 @@ func (m *SlowLog) Run(ctx context.Context) {
 				// FIXME https://jira.percona.com/browse/PMM-3898
 				m.l.Infof("Events channel is closed.")
 				running = false
-				m.changes <- Change{Status: inventorypb.AgentStatus_WAITING}
+				m.changes <- Change{Status: inventorypb.AgentStatus_RUNNING}
```

Comment:
My understand is that we are waiting for a restart there. Why RUNNING? Also, that does not make sense to do that right after `running = false` assignment.